ie-solver/helpers.h

```diff
@@ -3,25 +3,15 @@
 #define IE_SOLVER_HELPERS_H_
 
 #include <vector>
-#include <memory>
 #include <string>
-#include <fstream>
-#include "ie-solver/ie_mat.h"
 #include "ie-solver/ie_solver_config.h"
 #include "ie-solver/boundaries/boundary.h"
-#include "ie-solver/boundaries/circle.h"
-#include "ie-solver/boundaries/rounded_square.h"
-#include "ie-solver/boundaries/rounded_square_with_bump.h"
-#include "ie-solver/boundaries/squiggly.h"
-#include "ie-solver/boundaries/ellipses.h"
-#include "ie-solver/boundaries/cubic_spline.h"
 
-#define TEST_SIZE 100
 #define TIMING_ITERATIONS 5
 
 namespace ie_solver {
 
-double boundary_integral_solve(const ie_solver_config& config,
+double boundary_integral_solve(ie_solver_config& config,
```

Comment:
make config constant again - this will require come decent refactoring, probably having this function return the solution rather than the error, or something like that